### PR TITLE
Document alert_relabel_configs.

### LINKED ISSUE #568: Alert relabelling is undocumented

### PR DESCRIPTION
**Pull request by brian-brazil**
Fixes #568

### CODE REVIEW

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -714,10 +719,19 @@ the `replace`, `keep`, `drop` and `labelmap` actions. The regex is fully anchore
...
 
 One use for this is to blacklist time series that are too expensive to ingest.
+
+### `<alert_relabel_configs>`
+
+Alert relabeling is applied to alerts before they are sent to the Alertmanager.
+It has the same configuration format and actions as target relabeling. Alert
+relabeling is applied after external labels.
+
+One use for this is ensuring a HA pair of Prometheuis with different external
```
Comment (beorn7) [inline content/docs/operating/configuration.md]: "a HA" → "an HA" (depending on reading mode, though)
"Prometheis" (but honestly, I'd prefer "Prometheus servers").

**Comment by beorn7** [comment]
Otherwise, :+1:

File: content/docs/operating/configuration.md (hunk 1 above)
Comment (brian-brazil) [inline content/docs/operating/configuration.md]: An only before vowel or vowel like sounds, other changed.

Comment (juliusv) [inline content/docs/operating/configuration.md]: "HA" starts with a vowel sound in US English at least - "Aich-Aa". To me it reads really weird as "a HA" :)

Comment (beorn7) [inline content/docs/operating/configuration.md]: /ˈeɪtʃ/ or /ˈheɪtʃ/ , both possible, that's what I meant by "reading mode". The latter is more an Irish thing, see https://en.wikipedia.org/wiki/H#Name_in_English

**Pull request merged**